package.json

```diff
@@ -1,41 +1,37 @@
 {
```

Comment:
@baer okay, I added `lodash` and restored the `package.json` formatting.  That way I was also able to use lodash's `find` function instead of `.filter(...).shift()`.